configuration/ledger.go

```diff
@@ -52,6 +52,24 @@ type Exporter struct {
 	ExportLag uint32
 }
 
+// Replica holds configuration for Replicator.
+type Replica struct {
+	// Role defines that should do replicator (subscribe on replica parent or send notifications to replica targets).
+	Role string
```

Comment:
It's strange for me, that we have a part of observer functionality inside a heavy's node. Because observer is a standalone node